tests/certbot-boulder-integration.sh

```diff
@@ -280,23 +280,32 @@ CheckCertCount() {
     fi
 }
 
-CheckGroup() {
-    group_mode() { echo $((0`stat -c %a $1` & 070)); }
-    group_owner() { echo `stat -c %G $1`; }
-    if [ `group_mode $1` -ne `group_mode $2` ] ; then
-        echo "Expected group permission `group_mode $1`, got `group_mode $2` on file $2"
+CheckPermissions() {
```

Comment:
I'm not sure this function is working as we expect. See https://travis-ci.org/certbot/certbot/jobs/462933676#L4862. It's complaining that `$3` is unbound which I think is happening because `$3` is getting reset in the scope of the `masked_mode` function.

Reply:
Now it's complaining that it can't find the command `074` and `masked_mode` is always returning 0. The fix might be as simple as changing the backticks to quotes, but I didn't play around with it.

Reply:
I'll play around with it. Thanks for your patience on this-- i'm pretty green when it comes to shell scripting.

Reply:
No worries! I learned bash by working on Certbot.

I just quickly made a note of the problem because I was trying to get a few things done, but if you're spending much time trying to fix and would like some help, please let me know!

Reply:
Do not push too hard on bash tests, they are rewritten in Python with #6541. At worst, you could just describe what you want as a comment, and I will write it in Python once your PR is merged in master and that I sync my PR with master.